modules/util/portal-tools-rest-builder/src/main/resources/com/liferay/portal/tools/rest/builder/dependencies/base_resource_test_case.ftl

```diff
@@ -129,6 +130,14 @@ public abstract class Base${schemaName}ResourceTestCase {
 		testCompany = CompanyLocalServiceUtil.getCompany(testGroup.getCompanyId());
 
 		<#if generateDepotEntry>
+			irrelevantDepotEntry = DepotEntryLocalServiceUtil.addDepotEntry(
+				Collections.singletonMap(LocaleUtil.getDefault(), RandomTestUtil.randomString()),
```

Comment:
there is a ServiceContextRequestUtil that maybe we should extend with more creation methods to avoid the awful {{